### DISCUSSION TITLE
Adding variables to style attribute issue

### DISCUSSION THREAD

**Discussion by rrowinskiflaconi** [maizzle/framework#1099] [Q&A]
- Maizzle Version: 4.6.0
- Node.js Version: 16.20.2

I would like to add variables to style attribute without quotation marks (single quotes in this case '  ' ) but when I remove them the build fails

template
` <a style="color: '@{variables.categoryBgColor}'">
   click me
</a>`

output
` <a style="color: 'red'">
   click me
</a>`

setup
`  module.exports = {
      build: {
        posthtml: {
          expressions: {
            delimiters: ["{", "}"],
            unescapeDelimiters: ["{{{", "}}}"],
          },
        },
       },
      },`

**Comment by ayhanap** [comment]
Trying to do a similar thing. I want to render the following html. But the build complains with `Unknown word`. `{tag.color}` is a syntax for another templating engine.

`<a style="background-color: {tag.color}">click</a>`

**Comment by cossssmin** [comment]
It's PostCSS that's throwing on invalid CSS syntax.

I've been testing [postcss-safe-parser](https://github.com/postcss/postcss-safe-parser) for Maizzle 5, I try to add it in the current version as well.

Meanwhile, you can try installing and using it yourself:

```bash
npm install postcss-safe-parser
```

```js
module.exports = {
  build: {
    postcss: {
      plugins: [
        require('postcss-safe-parser')()
      ]
    }
  }
}
```

**Comment by ayhanap** [comment]
Thanks, I will give it a try. I managed to bypass it by using `<a style="background-color: <%tag.color%>">click</a>` then with a `afterTransformers` replacing `<%` with `{` atm.
